RussianFilter/sections/general_extensions.txt

```diff
@@ -350,6 +350,8 @@ segodnya.ua#$#iframe[id^="google_ads_iframe"] { visibility: hidden!important; ma
 film.ru#$#.b-nav-string { margin-top: 50px!important; }
 ! fasttorrent.su - удаление отступа сверху
 fasttorrent.su#$##container { margin: 64px auto 0 auto!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/31336
+26-2.ru#$#body { margin-top: 40px!important; }
```

Comment:
Maybe it would be better to add `26-2.ru#$#body.branded { margin-top: 40px!important; }`, so this rule would not be applied on mobile website?

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/59433857-a4e75b00-8dea-11e9-8a56-d5cffe6aa40f.png)

</details>